src/PassengerClass.js

```diff
@@ -0,0 +1,16 @@
+const { bookshelf, Passenger, Passengers, knex } = require('../database/connection.js')
+const commands = require('./commands.js')
+
+class PassengerClass {
+
+  getById(id){
+    return commands.getPassenger( {id} )
+  }
+
+  getByFirstName(FirstName){
```

Comment:
convention is to use lower, camel case for variable identifiers (`FirstName` looks like a class to me)